Microsoft.NET.Build.Containers/CreateNewImage.cs

```diff
@@ -94,7 +94,9 @@ public class CreateNewImage : Microsoft.Build.Utilities.Task
     /// </summary>
     public ITaskItem[] Labels { get; set; }
 
-    private bool IsDockerPush { get => OutputRegistry == "docker://"; }
+    private bool IsDockerPush { get => String.IsNullOrEmpty(OutputRegistry); }
+
+    private bool IsDockerPull { get => String.IsNullOrEmpty(BaseRegistry); }
```

Comment:
similar to my comment above, what do we expect to happen if the user manually sets `docker.io`?

Reply:
if they set docker.io then I'd expect to pull from the docker hub registry (e.g. I can have a container at docker.io/chusk3/sdk-container-demo, for example).